docs/architecture.dot

```diff
@@ -2,30 +2,35 @@
 # dot architecture.dot -Tsvg -oarchitecture.svg
 
 digraph G {
-	label="Last updated 2017-07-13"  # UPDATE WHEN CHANGED!
+	label="Kubernetes Test Infrastructure (Last updated 2020-03-04)"  # UPDATE WHEN CHANGED!
+    labelloc="t"
+    graph[compound=true]
 
 	# external services
-	legend_ext, GCS, PubSub, GitHub, BigQuery [style=filled, fillcolor=gray, shape=square]
+	legend_ext, GCS, PubSub, "GitHub\nthrough\nghproxy", BigQuery [style=filled, fillcolor=gray, shape=square]
 
 	# things that host websites
-	legend_site, Triage, Mungegithub, Deck, Testgrid, Gubernator [shape=house, fillcolor=lightblue, style=filled]
+	legend_site Spyglass, Deck, "Testgrid (closed)", Gubernator, Velodrome, Boskos, Greenhouse [shape=house, fillcolor=lightblue, style=filled]
```

Comment:
Velodrome is now a thing of the past, as well as the Influx AFAIK @spiffxp, hopefully we can replace them with BigQuery + DataStudio soon, I will remember to update this if that lands